src/django_csrf_fetch.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+/* global fetch */
+import 'isomorphic-fetch'
+import R from 'ramda'
+
+import { S, parseJSON, filterE } from './util'
+
+export function getCookie (name: string): string|null {
+  if (document.cookie && document.cookie !== '') {
+    let possibleCookies = document.cookie
+      .split(';')
+      .map(cookie => cookie.trim())
+      .filter(cookie => cookie.substring(0, name.length + 1) === `${name}=`)
+      .map(cookie => decodeURIComponent(cookie.substring(name.length + 1)))
```

Comment:
Does it make sense to use `find` here? It would skip the `possibleCookies[0]` part below but it would return `undefined` instead of `null`, not sure if that's significant